Assets/MixedRealityToolkit/UX/Scripts/Dialog/DialogResult.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace MixedRealityToolkit.UX.Dialog
+{
+    public class DialogResult
+    {
+        /// <summary>
+        /// The button press that closed the dialog
+        /// </summary>
+        public Dialog.ButtonTypeEnum Result = Dialog.ButtonTypeEnum.Close;
```

Comment:
can this be private? If so please make the change and add the serialized attribute.
If not then add a public property that utilizes the field as a private serialized field.